Doc/reference/compound_stmts.rst

```diff
@@ -811,23 +834,26 @@ able to suspend execution in its *enter* and *exit* methods.
 
 The following code::
 
-    async with EXPR as VAR:
-        BLOCK
+    async with expression as target:
```

Comment:
What is the purpose of this change?

Reply:
@serhiy-storchaka To use the same terms as in the grammar.

Reply:
It's not important to match the grammar in this case as the docs are not kept in sync with the grammar rule names.

Reply:
Right, we deliberately use ALL_CAPS in these semantic expansions to distinguish the placeholders from keywords and hidden variables.

Reply:
@ncoghlan All right, updated.